backend/src/main/java/com/bakdata/conquery/models/auth/util/SinglePrincipalCollection.java

```diff
@@ -1,110 +0,0 @@
-package com.bakdata.conquery.models.auth.util;
```

Comment:
Datei wird mir nicht als gelöscht sondern nuur leer angezeigt

Reply:
you right :)

Reply:
sollte jetzt weg sein

Reply:
![image](https://user-images.githubusercontent.com/1553491/140047879-f5b9caf4-3332-44b1-8dc0-fd557fab3743.png)

Reply:
Komischer intellij merge